sdk/quantum/Microsoft.Azure.Management.Quantum/tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using Xunit;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Quantum.Tests")]
```

Comment:
Can you take a look compute's assemblyinfo and fill in Microsoft specific text?

Reply:
Done.
I wasn't sure what to put on the `assembly: Guid` attribute and I saw that other tests projects are using the same guid from the Compute.Tests project, so I put the same.
Since this is not exposed to COM it shouldn't matter, right?